### ISSUE TITLE
How can we better map 4:3 games to 16:9 monitors?

### ISSUE THREAD

**Issue by walikor** [openMSX/openMSX#1392]
hi,
i have a 16: 9 monitor with 2560X1440 resolution, but in full screen it is like 4: 3, is there any way to fit it? i tried to select all options on Horizontal Stretch but they don't work.
Thanks

https://ibb.co/fFGC076

**Comment by MBilderbeek** [comment]
There is no way around this. This originates from the 4:3 aspect ratio the MSX video output has, and thus a 4:3 monitor will do that justice... Otherwise your video will be horizontally deformed.

**Comment by mthuurne** [comment]
I agree with Manuel that 4:3 is the correct aspect ratio for MSX games. So I'd like to rephrase the problem as "how can we better map 4:3 games to 16:9 monitors"?

There are some things we could do, like having a configurable vertical overscan, so you could opt to reduce the top/bottom borders to use more of the screen space for the display area.

Something that is often done to make a 4:3 video fit a 16:9 frame is to stretch and heavily blur the video and put that on the sides instead of black bars, while leaving the center video in the original 4:3 aspect ratio.

There are some more sophisticated hacks possible, like [wideNES](https://prilik.com/blog/post/widenes/), but there aren't all that many horizontally scrolling games on MSX, so I'm not sure that's worth putting a lot of effort into.

Another option might be to overlay the status area (where it lists score, lives etc.) of a game on top of the game area. However, identifying the status area would require either complicated image recognition or per-game configuration, so that's not a simple solution either.

**Comment by Vampier** [comment]
it's a setting on the TV
